prime-router/metadata/HL7/v251-elr/hl7/segments/ORC/Specimen.yml

```diff
@@ -19,7 +19,7 @@ collection:
         dateTimeIn: OBR.7
         dateTimeOut: OBR.8
       expressions:
-        - valueOf: ../../datatypes/DR/Period
+        - valueOf: ../../datatypes/v251/hl7/datatypes/DR/Period
```

Comment:
Noticed this was off when triaging why the new nist elr java class was not working